Muzikie/service
# Add support for indexing audio reclaim command

### What was the problem?
This PR resolves #26 

### How was it solved?
 - This PR must be merged after the audio:stream PR. It is based on said PR and the merge order matters.
 - I'm updating the audio income value, and since I use the token transfer method (in the blockchain) to transfer the income to the artist account, we don't have to worry about that in Streamer, it is already handled by the token module.


### How to test it
- Create a collection, and the an audio
- Stream the audio
- Reclaim the income
- Check the balances in each step to spot the changes.
